graphql/requirements-dev.txt

```diff
@@ -2,7 +2,7 @@ uvicorn~=0.18.3
 strawberry-graphql[debug-server]==0.133.5
 fastapi==0.85.0
 pydantic==1.10.2
-SQLAlchemy==1.4.11
+SQLAlchemy==2.0.5
```

Comment:
Given the newness of this, might be a good time to switch to `SQLAlchemy=~2.0.5` here and elsewhere. Would be good to automatically pull in bugfixes.

Reply:
Does using `=~` instead of `==` automatically pull in changes to a version?

Reply:
updated